Workbooks/Usage/User Impact Analysis/User Impact Analysis.workbook

```diff
@@ -140,7 +94,7 @@
             "type": 1,
             "description": "The query is used by ByValue, SplityBy and Properties dropdown",
             "isRequired": true,
-           "query": "union customEvents, pageViews, requests\r\n            {limitedTimeRange}\r\n            | project cols = column_ifexists(\"customDimensions\", dynamic(null))\r\n            | extend usage_colkeys = tostring(bag_keys(cols))\r\n            | where usage_colkeys !has '*'\r\n            | evaluate bag_unpack(cols)\r\n            | getschema \r\n            | top-nested 100 of ColumnName by Count = count() desc\r\n             | extend ColumnName = replace(@\"\\\\\", @\"\\\\\\\\\",ColumnName) \r\n            | project value =strcat(\"customDimensions[\\\"\", replace(@'\"',@'\\\\\\\\\"', ColumnName), \"\\\"]\"), label =replace(@'\"',@'\\\\\"', ColumnName), group='Custom Properties'\r\n            | order by label asc\r\n            | project data = strcat(\"'\", replace(@\"\\'\",@\"/:/\", value),\"','\",replace(@\"\\'\",@\"/:/\", label),\"','\",group,\"'\")",
+           "query": "let endTime = {TimeRange:end};\r\nlet startTime =  iif(endTime - 7d > {TimeRange:start}, endTime - 7d, {TimeRange:start});\r\nunion customEvents, pageViews, requests\r\n            | where timestamp between(startTime..endTime)\r\n            | project cols = column_ifexists(\"customDimensions\", dynamic(null))\r\n            | extend usage_colkeys = tostring(bag_keys(cols))\r\n            | where usage_colkeys !has '*'\r\n            | evaluate bag_unpack(cols)\r\n            | getschema \r\n            | top-nested 100 of ColumnName by Count = count() desc\r\n             | extend ColumnName = replace(@\"\\\\\", @\"\\\\\\\\\",ColumnName) \r\n            | project value =strcat(\"customDimensions[\\\"\", replace(@'\"',@'\\\\\\\\\"', ColumnName), \"\\\"]\"), label =replace(@'\"',@'\\\\\"', ColumnName), group='Custom Properties'\r\n            | order by label asc\r\n            | project data = strcat(\"'\", replace(@\"\\'\",@\"/:/\", value),\"','\",replace(@\"\\'\",@\"/:/\", label),\"','\",group,\"'\")",
```

Comment:
this is change is just adding what criteria params did into the query?

Reply:
Yes and remove useless parameters